### DISCUSSION TITLE
Session-id-global

### DISCUSSION THREAD

**Discussion by nguyenduchoa37** [rtbrick/bngblaster#137] [Q&A]
{session-global}: As I understand, this is ID for each subscribers, and always begins from 1. But if I have 10K subscribers in AAA, from subscribers_1 to subscriber_10000, and I want to have to bngblaster sessions for them, and each session is for 5K. How can I choose username in json config for session from 5000 to 10000?

**Comment by GIC-de** [comment]
You need to use the variables `{i1}` or `{i2}` which can be freely configured (https://rtbrick.github.io/bngblaster/interfaces.html#access-interfaces):

The configuration attributes for username, password, agent-remote-id, agent-circuit-id, and cfm-ma-name support variable substitution. The variable {session-global} will be replaced with the actual session-id starting from 1 and incremented for every new session. The variable {session} is incremented per-interface section. The variables {outer-vlan} and {inner-vlan} will be replaced with the corresponding VLAN identifier or 0 if not defined. The two variables {i1} and {i2} are configurable per-interface sections with user-defined start values and steps.
